era_5g_relay_network_application/worker_subscriber.py

```diff
@@ -135,9 +143,13 @@ def callback(self, data: Any):
 
                 self.queue.put_nowait((sid, msg))
             else:
+                if self.memory is not None:
+                    print(f"storing from {self.topic_name}")
```

Comment:
remove

Reply:
Doesn't seem as done :-) Or do you want to keep there `print`?

Reply:
Oh, sorry, forgot to save the file :-D now it has been removed